packages/icons/src/Icon/ColoredIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Rename `Icon` folder to `components`

Please add both of these files native suffix `ColoredIcon.native.tsx` and create empty component  `ColoredIcon.tsx` `() => null` with comment that it will be used for web too in future. Also please mention that in README.